src/lib/builder/trace-hint-builder/index.ts

```diff
@@ -113,6 +106,7 @@ class TraceHintBuilderClass {
       pcb_port_id: pcb_port.pcb_port_id,
       pcb_component_id: pcb_port.pcb_component_id,
       route,
+      trace_width: this.props.traceWidth ?? 0.1,
```

Comment:
we shouldn't give trace hints a default width

Reply:
(because we want to have the priority system- if a trace hint width isn't defined then we'll use the default for the net, or the pad type etc.)

Reply:
What is the difference between the net and pad type? And how are they coming into the picture while we are manually adding trace hint?

Reply:
Yea good question, a "net" is really generally, like "the 5v net" usually refers to any connection that makes contact with a 5v line. Usually that's going to be like every chip that has power. Lowest priority for trace width 

Then there's a trace, which refers to the full set of lines/route that connect two or more pads together.

Then theres a "tracehint point" which refers to a single point along a tracehint. Each of these points can individually have an associated width.

Will draw up a diagram in a moment

Reply:
![image](https://github.com/user-attachments/assets/2b29fcd6-ca12-4a27-886b-c70f5a19d2dd)

Reply:
Is trace only between the chip and power? then for ex. what are those between two resistors or a chip and resistor?

We use these trace_hints to make a optimised trace and optimised width usage for trace?

Reply:
generally you don't need to vary the trace width for non-power traces, but a circuit will often have 2-5 different traces that represent some kind of power at different voltage levels or "noise levels"

Not every trace will have an explicitly defined net too. If you don't have an explicit net then you just use the "board's default trace width". An explicit net is a net that's actually written in the code. Most nets are not explicit, they're created "on the fly" or "as needed"

A trace that isn't connected to power is generally called a "signal trace" because it carries either data or a signal related to data.